import-brage/build.gradle

```diff
@@ -4,18 +4,14 @@ plugins {
 
 version 'unspecified'
 
-repositories {
-    mavenCentral()
-}
 
 dependencies {
     def jacksonVersion = '2.12.0'
-    implementation group: 'jakarta.xml.bind', name: 'jakarta.xml.bind-api', version: '2.3.2'
+    implementation group: 'com.github.BIBSYSDEV', name: 'nva-datamodel-java', version: '0.11.1'
```

Comment:
can you put this versionas an project.ext variable so that it is easier to spot on upgrades?